SCClassLibrary/Common/GUI/Base/QDialog.sc

```diff
@@ -17,7 +17,7 @@ FileDialog : QObject {
 			false: okFunc(paths)
 		**/
 
-		var me = super.new( [fileMode, acceptMode] );
+		var me = super.new( [fileMode, acceptMode, path] );
 
 		if( okFunc.notNil ) {
 			me.connectFunction( 'accepted(QVariantList)', {
```

Comment:
remove space:
`me.connectFunction('accepted(QVariantList)', {`

Reply:
i haven't touched this line in this branch